price/fixed_method_pricing_test.go

```diff
@@ -0,0 +1,37 @@
+package price
+
+import (
+	"github.com/singnet/snet-daemon/blockchain"
+	"google.golang.org/grpc"
+
+	"math/big"
+	"testing"
+
+	"github.com/singnet/snet-daemon/handler"
+	"github.com/stretchr/testify/assert"
+)
+
+
+
+func TestFixedMethodPrice_initPricingData(t *testing.T) {
+	metadata,_ := blockchain.ReadServiceMetaDataFromLocalFile("../service_metadata_method_pricing.json")
```

Comment:
I would move test data into package and rename it to make clear that it is test data. Even better solution would be using test string instead of external file.

Reply:
I will use test string , ( Dont  want to mix it with the files set up for local testing )

Reply:
also moved the new json file to a separate testing folder ( useful if someone wants to quickly see how the new pricing json looks like and wants to test this end to end locally )